tests/installation/partitioning_warnings.pm

```diff
@@ -100,7 +100,7 @@ sub run {
     }
     # create small enough partition (11GB) to get warning for enabled snapshots
     # on storage-ng snaphots are disabled as per proposal when partition is to small, so enable to check the warning
-    addpart(role => 'OS', size => 11000, format => 'btrfs', enable_snapshots => is_storage_ng);
+    addpart(role => 'OS', size => (is_opensuse) ? 9000 : 11000, format => 'btrfs', enable_snapshots => is_storage_ng);
```

Comment:
Could you explain me why openSUSE is different here?

Reply:
Thank you for updating the comments, I have miss this part. I am not really aware how the algorithm works so I have been following the warnings [leap15.1 - 10GB](https://openqa.opensuse.org/tests/764759#step/partitioning_warnings/37) and for [sle15sp1 - 12 GB](https://openqa.suse.de/tests/2095938#step/partitioning_warnings/37).
Also there is a different threshold for small root [leap15.1 - 5GB](https://openqa.opensuse.org/tests/764759#step/partitioning_warnings/148) and [sle15sp1 - 3GB](https://openqa.suse.de/tests/2095938#step/partitioning_warnings/148)

Reply:
@DimStar77 @lnussel @fcrozat is the difference in the root partition limit under which no snapshots are proposed among TW/Leap/SLE on purpose? I guess these are settings in control.xml files?

Reply:
> @DimStar77 @lnussel @fcrozat is the difference in the root partition limit under which no snapshots are proposed among TW/Leap/SLE on purpose? I guess these are settings in control.xml files?
Yes, it is part of each distro control.xml or even the system roles, these days.